test/bench/100_pods/S3/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#lunch runs with the same nb pods
+# warm up run
+#./bench_10k.sh
+
+#clearmeasured runs
+for i in {1..3}
+do
+    ./bench_10k.sh >> 10k.json
+done
+
+#clean the output file
+python3 cleaner.py
```

Comment:
ne fonctionnera que sur ubuntu

sudo apt-get install python3-is-python

Reply:
Encore mieux : mettre un shebang, et appeler le script comme un programme.

Reply:
Done !